website/docs/manage_groups.md

```diff
@@ -51,4 +60,59 @@ More levels of folder nesting are not supported.
 │   └─Japanese   <- Group
 |       └─DictB  <- Dict Files's container folder
 |          └─ DictB Files  
-```
\ No newline at end of file
+```
+
+## group by metadata.toml
+
+Click the "group by metadata" will group your dicts based on metadata.toml file.
+
+the location of the metadata file is at the same location as the dictionary.
+
+The structure of the metadata file use [toml](https://toml.io/en/) format
```

Comment:
Just `https://toml.io` 

The website has a Chinese version, browser may automatically use it.

Reply:
![image](https://user-images.githubusercontent.com/105986/235301741-dc1dccac-4373-4475-b3e0-a455fa43844e.png)

this is a feature of github ,you can recommend code changes directly and I can just apply them.